cmd/beacon-chain/flags/base.go

```diff
@@ -53,6 +53,16 @@ var (
 		Name:  "tls-key",
 		Usage: "Key for secure gRPC. Pass this and the tls-cert flag in order to use gRPC securely.",
 	}
+	// DisableHTTPPrysmAPI disables all HTTP endpoints of Prysm API.
+	DisableHTTPPrysmAPI = &cli.BoolFlag{
+		Name:  "disable-http-prysm-api",
+		Usage: "Disable all HTTP endpoints of Prysm API",
+	}
+	// DisableHTTPEthAPI disables all HTTP endpoints of the official Ethereum API.
+	DisableHTTPEthAPI = &cli.BoolFlag{
+		Name:  "disable-http-eth-api",
+		Usage: "Disable all HTTP endpoints of the official Ethereum API",
+	}
```

Comment:
Can we follow the same model that geth uses? They use a flag like `--http.api=eth,admin,debug`. On the code side, these are http rpc modules, with the default being "eth" namespace. 

https://github.com/ethereum/go-ethereum/blob/108eec3feeca2e05d3daf1a198b8286623f56755/cmd/geth/config.go#L114

Then they call RegisterApis which only exposes the user defined or default modules.

https://github.com/ethereum/go-ethereum/blob/108eec3feeca2e05d3daf1a198b8286623f56755/node/rpcstack.go#L283